app/routes/user/password.tsx

```diff
@@ -105,31 +119,21 @@ export function ResetPassword() {
           <Label htmlFor="oldPassword">Old Password</Label>
           <TextInput
             data-focus
+            name="oldPassword"
             className={
-              errorMessage && !interacted
-                ? 'usa-input--error'
-                : 'margin-bottom-4'
+              !isOldPasswordTouched && errorMessage ? 'usa-input--error' : ''
             }
-            name="oldPassword"
             id="oldPassword"
             type="password"
             placeholder="Old Password"
             onChange={(e) => {
               setOldPassword(e.target.value)
               setInteracted(true)
+              setIsOldPasswordTouched(true)
             }}
           />
-          {errorMessage === 'NotAuthorizedException' && !interacted && (
-            <div className="text-red margin-bottom-105 margin-top-2px">
-              Invalid Password
-            </div>
-          )}
-          {errorMessage === 'LimitExceededException' && !interacted && (
-            <div className="text-red margin-bottom-105 margin-top-2px">
-              Attempts Exceeded. Please try again later.
-            </div>
-          )}
-          <Label htmlFor="newPassword" className="margin-top-0">
+          {ErrorMessage(errorMessage, interacted)}
```

Comment:
Use JSX syntax here:

```suggestion
          <ErrorMessage errorMessage={errorMessage} interacted={interacted} />
```